src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -30,6 +31,9 @@ public function __construct(Connection $connection)
         $this->connection = $connection;
     }
 
+    /**
+     * @return static
```

Comment:
On second look, this one is rather a mistake : the base interface doesn't say it should return anything, and the other methods don't. The right fix would be to make this `@return void`... I doubt we can create a nice BC layer for that, but let's still add the `@return void` so that SA can know what to expect?